src/infra/zookeeper-deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
@amerkul consider zookeeper as `StatefulSet`, not the `Deployment`

since zookeeper has state, it's not _stateless_